code/game/mecha/equipment/tools/tools.dm

```diff
@@ -49,7 +50,7 @@
 			if(locate(/mob/living) in O)
 				occupant_message(SPAN_WARNING("You can't load living things into the cargo compartment."))
 				return
-			if(O.anchored)
+			if(O.anchored &! allow_pickup)
```

Comment:
This hurts so much I had to call someone else to review it for me.

Reply:
Trilbyspaceclone, what you've just coded is one of the most insanely idiotic things I have ever read. At no point in your rambling, incoherent pull request were you even close to anything that could be considered a functional piece of code. Everyone in this GitHub repo is now dumber for having seen it. I award you no points, and may God have mercy on your soul.

Reply:
I was shown a screenshot of this and as the person that largely rewrote mechs for tg my eyes are now stained why would you do  this to me